devel/build

```diff
@@ -48,7 +48,9 @@ docker buildx build \
     --build-arg CACHE_DATE \
     --build-arg GIT_REVISION \
     --cache-from $BASE_BUILDER_IMAGE:latest \
+    --cache-from $BASE_BUILDER_IMAGE:$tag \
     --cache-from $BASE_IMAGE:latest \
+    --cache-from $BASE_IMAGE:$tag \
     --cache-to type=inline \
```

Comment:
Is this at all problematic when `$tag == latest`?

Reply:
No, I just ran `./devel/build` locally and got this without error:

```
 => importing cache manifest from nextstrain/base-builder:latest                                                                                                                                                      1.0s
 => importing cache manifest from nextstrain/base:latest   
```